homeassistant/components/attribute_as_sensor/strings.json

```
@@ -0,0 +1,42 @@
+{
+  "title": "Attribute as Sensor",
+  "config": {
+    "step": {
+      "user": {
+        "title": "Add Attribute as Sensor",
+        "description": "Create a sensor that derives it's value from an attribute from another entity.",
+        "data": {
+          "entity_id": "Input entities",
+          "name": "Name",
+          "icon": "Icon"
+        }
+      },
+      "attr": {
+        "data": {
+          "attribute": "Attribute",
+          "device_class": "Device Class",
+          "state_class": "State Class",
+          "unit_of_measurement": "Unit of Measurement"
+        },
+        "data_description": {
+          "attribute": "Select the Attribute from the entity that you like to get as sensor.",
+          "unit_of_measurement": "Select an unit of measurement or add your own custom UoM"
```

```suggestion
          "unit_of_measurement": "Select a unit of measurement or add your own custom UoM"
```
English is weird https://www.pristineword.com/grammar-a-unit/#:~:text=pronunciation%2C%20not%20spelling.-,It's%20a%20unit.,the%20indefinite%20article%20%E2%80%9Ca%E2%80%9D.

---

Probably yes, too little focus 😀